tests/posix/threads_ext/src/main.c

```diff
@@ -466,7 +466,7 @@ ZTEST(pthread_attr, test_pthread_attr_getscope)
 	zassert_equal(contentionscope, PTHREAD_SCOPE_SYSTEM);
 }
 
-ZTEST(pthread_attr, test_pthread_attr_setscope)
+ZTEST(posix_threads_ext, test_pthread_attr_setscope)
```

Comment:
I believe this is part of POSIX_THREADS_BASE and likely not POSIX_THREADS_EXT

But it does require the CONFIG_POSIX_THREAD_PRIORITY_SCHEDULING option.